astro/src/components/RemoteContent.astro

```diff
@@ -1,8 +1,9 @@
 ---
 /**
-  * Remote content fetcher. Remote content is assumed to be markdown.
-  */
-import { marked } from 'marked';
+ * Remote content fetcher. Remote content is assumed to be markdown.
+ */
+import { Markdown } from "astro-remote";
```

Comment:
This is not found, did you include this in the package.json?

Reply:
my bad @lyleschemmerling I think I installed before I created the new branch.